src/main/java/com/github/fabienrenaud/jjb/support/BenchSupport.java

```diff
@@ -60,7 +61,8 @@
             new Libapi(Library.TAPESTRY),
             new Libapi(Library.JSONITER, Api.DATABIND),
             new Libapi(Library.MINIMALJSON),
-            new Libapi(Library.UNDERSCORE_JAVA)
+            new Libapi(Library.UNDERSCORE_JAVA),
+            new Libapi(Library.PUREJSON)
```

Comment:
Stream serializer & deserializers are not implemented for purejson + client model.
Remove.